obspy/signal/tests/test_sonic.py

```diff
@@ -4,6 +4,12 @@
                         unicode_literals)
 from future.builtins import *  # NOQA
 
+from obspy import Trace, Stream, UTCDateTime
```

Comment:
These imports almost all exist a little further down (and should be there, not here.)